<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLightPoint</class>
 <widget class="QWidget" name="frmLightPoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmLightPoint</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="LightPoint" name="lightPoint1" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="LightPoint" name="lightPoint2" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="LightPoint" name="lightPoint3" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="LightPoint" name="lightPoint4" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="LightPoint" name="lightPoint5" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="LightPoint" name="lightPoint6" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LightPoint</class>
   <extends>QWidget</extends>
   <header location="global">lightpoint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
